module.js

```diff
@@ -110,6 +110,9 @@ const initSync = (bytes) => {
  */
 const init = async (input) => {
   if (typeof input === "undefined") {
+    // TODO: using `import.meta` like this does not work in cjs.
```

Comment:
Yes, this is correct. We could give it the same treatment we use in `main.js` I think?